.github/workflows/canary-integration-test.yml

```diff
@@ -1,39 +1,24 @@
-name: Canary integration tests
+name: Reusable canary integration tests
 on:
-  push:
-    tags:
-      - v*
-    branches:
-      - master
-      - release-*
-  pull_request:
-    branches:
-      - master
-      - release-*
-    paths-ignore:
-      - "Documentation/**"
-      - "design/**"
   workflow_call:
     inputs:
-      ceph-image:
-        description: 'Ceph image for creating Ceph cluster'
-        default: 'quay.io/ceph/ceph:v18'
+      ceph_images:
+        description: 'JSON list of Ceph images for creating Ceph cluster'
+        default: '["quay.io/ceph/ceph:v18"]'
         type: string
```

Comment:
`${{ inputs.ceph_images }}` is not available `on.pull_request` or `on.push`, so this file is now **only** the reusable `on.workflow_call` portion that is invoked from other files.